src/RA_httpthread.cpp

```diff
@@ -431,6 +430,15 @@ void HttpResults::PushItem(RequestObject* pObj)
     ReleaseMutex(RAWeb::Mutex());
 }
 
+void HttpResults::PushItem(std::unique_ptr<RequestObject> pObj)
```

Comment:
`unique_ptr `can't be copied it's passed by value on purpose, a move will be forced to transfer ownership.